docs/source/theory/windEnergy.rst

```diff
@@ -168,6 +168,119 @@ stabilization approaches are documented in
 to read those sections to gain a thorough understanding of the discretization
 scheme and the its impact on the simulations.
 
+Time stepping scheme
```

Comment:
This is an old style, circa 2006 method by which the algorithm is described, e.g., * and ** usage, along with the block notation to describe splitting and stabilization errors.. Moreover, the core algorithm is not a predictor-corrector method. I think casting it to a pressure projection scheme is more appropriate especially since P/C algorithms generally imply some sort of advanced predictor, e.g., A/B, prior to even the first provisional momentum solve. I think that I provided these comments to @gantech about six months ago and then @rcknaus followed up with him.

I need to work through this language, however, will probably not get to it until after the year. As such, feel free to push and we can negotiate the text changes next year. Since this is wind-energy specific, we can probably have different opinions on how we describe the methods.